tests/utils_test.py

```diff
@@ -0,0 +1,40 @@
+"""Unit tests for english language checker"""
+from libcipher import utils as utils
+import io
+import sys
+import unittest
+
+
+def get_marys_dictionary():
+        english_words = ["had", "a", "little", "lamb"]
+        text = str("\n".join(english_words))
+        if sys.version < '3':
+            dict_file = io.StringIO(unicode('\n').join(english_words))
+        else:
+            dict_file = io.StringIO("\n".join(english_words))
```

Comment:
@bravegnu Removing the unicode function caused the build to fail for python 3.2 I do not know why this happens. Can i try this any other way?
Thank you